### DISCUSSION TITLE
Github Student Pack features dont become available

### DISCUSSION THREAD

**Discussion by adema2003** [community/community#123736] [GitHub Education]
### Hi everyone,

Hi, I've submitted an application to receive the GitHub student pack. I've been a verified student since 2021 which you can see on the screenshot below
![Scherm­afbeelding 2024-05-13 om 10 51 56](https://github.com/community/community/assets/18036063/f5c7d133-297f-452e-999d-9da6c1f8a17a)

Yet when I check on the availability of the student pack I see the following:
![Scherm­afbeelding 2024-05-13 om 10 54 15](https://github.com/community/community/assets/18036063/ec59681d-e6bc-486f-8ead-4158bf52f65b)

Is there anyone that can look into this for me or knows how to fix this?

Thanks in advance,

Thomas

**Comment by nikhilsbuilds** [comment]
It seems you are facing an issue where GitHub is not recognizing your student verification status correctly when trying to access the GitHub Student Pack benefits.

A few suggestions that may help resolve this:

- Double-check that your student verification is still active and up-to-date on your GitHub account. Sometimes verifications can expire if not renewed.
- Try logging out of your GitHub account and logging back in, then attempt to access the Student Pack again.
- Clear your browser cache and cookies, then revisit the Student Pack page.
- If the issue persists, you may need to reach out to GitHub Support directly. They have a dedicated Student Pack support form at https://support.github.com/contact/education-form where you can explain the issue and provide any relevant screenshots.

GitHub's support team would be able to better investigate your specific account situation. Let me know if providing the screenshot is still an issue, and I'll try my best to further troubleshoot based on the information available.
